packages/react-icons/convert.js

```diff
@@ -93,17 +93,17 @@ function processFolder(srcPath, destPath, resizable) {
   var svgrOpts = {
     template: fileTemplate,
     expandProps: false, // HTML attributes/props for things like accessibility can be passed in, and will be expanded on the svg object at the start of the object
-    svgProps: { className: '{className}'}, // In order to provide styling, className will be used
-    replaceAttrValues: { '#212121': '{primaryFill}' }, // We are designating primaryFill as the primary color for filling. If not provided, it defaults to null.
+    svgProps: { className: '{undefined}'}, // In order to provide styling, className will be used
+    replaceAttrValues: { '#212121': '{undefined}' }, // We are designating primaryFill as the primary color for filling. If not provided, it defaults to null.
     typescript: true,
     icon: true
   }
 
   var svgrOptsSizedIcons = {
     template: fileTemplate,
     expandProps: false, // HTML attributes/props for things like accessibility can be passed in, and will be expanded on the svg object at the start of the object
-    svgProps: { className: '{className}'}, // In order to provide styling, className will be used
-    replaceAttrValues: { '#212121': '{primaryFill}' }, // We are designating primaryFill as the primary color for filling. If not provided, it defaults to null.
+    svgProps: { className: '{undefined}'}, // In order to provide styling, className will be used
```

Comment:
What is the overall effect of setting this undefined? Does it mean that people can no longer use classes to style/color the icons?

Reply:
`classname` and `fill` are still being set, they are just not being set using svgo. This allows for the svg to be directly passed into the `wrapIcon` method without having to explicitly define those values. They will then be defined in the `wrapIcon` method